src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1669,6 +1671,22 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    void TerminalPage::_ScrollHome(ScrollDirection scrollDirection)
+    {
+        auto indexOpt = _GetFocusedTabIndex();
+        // Do nothing if for some reason, there's no tab in focus. We don't want to crash.
+        if (!indexOpt)
+        {
+            return;
+        }
+
+        if (auto terminalTab = _GetTerminalTabImpl(_tabs.GetAt(*indexOpt)))
+        {
+            auto scrollDelta = _ComputeScrollDelta(scrollDirection, INT_MAX);
+            terminalTab->Scroll(scrollDelta);
+        }
```

Comment:
```suggestion
        if (const auto indexOpt = _GetFocusedTabIndex())
        {
            if (auto terminalTab = _GetTerminalTabImpl(_tabs.GetAt(*indexOpt)))
            {
                auto scrollDelta = _ComputeScrollDelta(scrollDirection, INT_MAX);
                terminalTab->Scroll(scrollDelta);
            }
        }
```
nit. You don't actually have to check `!indexOpt` and return early. You could instead reverse the logic of the if-statement and wrap the second one inside it 😉